### DISCUSSION TITLE
UI scale

### DISCUSSION THREAD

**Discussion by albmef** [amsynth/amsynth#230] [Q&A]
Hi !
any way to change the UI scale ?
i'm using amsynth on a raspberrypi 4 with à 10" touch screen (1280x800)
i wish i had a bigger UI, 125% scale would have been great
best regards
AM

**Comment by nickdowell** [comment]
There are currently two ways to scale the UI;
- Environment variable: `GDK_SCALE=2 amsynth`
- Command-line option: `amsynth --force-device-scale-factor 2`

Arbitrary rescaling is not currently implemented; there's an open feature request for this: 
- https://github.com/amsynth/amsynth/issues/29
